app/views/step_by_step_pages/schedule.html.erb

```diff
@@ -77,9 +77,15 @@
         width: "narrow",
       } %>
 
-      <div class="form-group">
-        <%= submit_tag "Schedule to publish", class: "btn btn-primary" %>
-        <%= link_to "Cancel", @step_by_step_page %>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+          <%= render "govuk_publishing_components/components/button", {
+            text: "Schedule to publish",
+          } %>
+        </div>
+        <div class="govuk-grid-column-full govuk-!-margin-top-3">
+          <%= link_to "Cancel", step_by_step_page_publish_or_delete_path(@step_by_step_page) %>
```

Comment:
Why has the destination of the `Cancel` link changed?

Reply:
I think the URL was originally `@step_by_step_page` when we had 'Scheduled' as its own tab item. We then moved it to a button on publish-or-delete.html but didn't change the URL.

It now feels a bit weird, in that if you click to 'Publish or delete', then 'Schedule to publish', then 'Cancel', you're not taken back to the 'Publish or delete' page.

Good spot - I'll move it into a separate commit.

Reply:
Have now split into two commits 👍